tests/codegen/sanitizer/riscv64-shadow-call-stack.rs

```diff
@@ -0,0 +1,16 @@
+//@ compile-flags: --target riscv64imac-unknown-none-elf -Zsanitizer=shadow-call-stack
+//@ needs-llvm-components: riscv
+
+#![allow(internal_features)]
+#![crate_type = "rlib"]
+#![feature(no_core, lang_items)]
+#![no_core]
+
+#[lang = "sized"]
+trait Sized {}
+
+// CHECK: ; Function Attrs:{{.*}}shadowcallstack
```

Comment:
You probably want a check line for the definition of `foo` and the attribute `#0` to match up w/ the check below.

Reply:
Added. Now the test is matching `foo` with `#0` attribute.